framework/src/utils/FunctionMaterialPropertyDescriptor.C

```diff
@@ -51,6 +51,17 @@ FunctionMaterialPropertyDescriptor::FunctionMaterialPropertyDescriptor(
 {
 }
 
+FunctionMaterialPropertyDescriptor::FunctionMaterialPropertyDescriptor(
+    const FunctionMaterialPropertyDescriptor & rhs, MooseObject * parent)
+  : _fparser_name(rhs._fparser_name),
+    _base_name(rhs._base_name),
+    _dependent_vars(rhs._dependent_vars),
+    _derivative_vars(rhs._derivative_vars),
+    _value(NULL),
```

Comment:
NULL? What is this, 2003?

Reply:
Dude, I've been writing 6502 assembly off work the past few weeks. This all is science fiction to me!

Reply:
FYI, those files were written in 2015(!). I've just pushed a commit to update them. There might be more to do in the files you didn't comment on...